runner/tasks.py

```diff
@@ -37,8 +37,11 @@
 from runner.cache.github_cache import GithubCache
 from lib.logger import format_log
 from lib.memcache_lock import memcache_task_lock
+from study.objects import StudyObject
+from study.models import JobGroupWatcher, JobGroupWatcherConfig
 from django.http import HttpResponse
```

Comment:
@buehlere I'm updating `develop` branch with `release/1.76.0`.

This line caused a conflict - I think this is for your `construct_csv()` changes. It was colliding with the StudyObject and JobGroupWatcher* imports.

I added it back in, but if you have problems let me know.